package.json

```diff
@@ -18,6 +18,7 @@
     "precompile": "npm run clean-lib && npm run copy-files && npm run babel",
     "prepublishOnly": "node check-version.js && npm install && npm run precompile",
     "watch": "npm run clean-lib && npm run copy-files -- --watch & npm run babel -- --watch",
+    "storybook": "./node_modules/.bin/start-storybook -p 9001 -c .storybook",
```

Comment:
does the `./` syntax work on windows?

Reply:
Yes it works.
![image](https://user-images.githubusercontent.com/12142840/52866483-0d754280-313f-11e9-95d4-ce0474d1c27e.png)